src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
         verticalPosition: 'top'
       });
     });
-  }
 
+    // Keyboard Shortcuts Listener
+    this.unlisten = this.keyboardShortcuts.listen(
+      {
+        // Navigation Page
+        'Control.Shift.n': (event: KeyboardEvent): void => {
+          this.router.navigate(['navigation']);
+          event.preventDefault();
+        },
+
+        // Checker Inbox & Pending Tasks
+        'Control.Shift.i': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
```

Comment:
Please remove any console log statements and other comments. Just mention:
`// TODO: Update once the route is created` in all cases where required.